hack/manifests/backstage/app-config.yaml

```diff
@@ -1,6 +1,10 @@
 app:
   title: Scaffolded Backstage App
   baseUrl: http://0.0.0.0:3000
+parodos:
```

Comment:
what is this configuration responsible for?

Reply:
it is to define the workflow and its tasks for assessment. Currently it has to be configured in this way. in the future we will have a controller to fetch it from backend